app/src/screens/passport/PassportNFCScanScreen.web.tsx

```
@@ -0,0 +1,59 @@
+import TextsContainer from '../../components/TextsContainer';
+import { BodyText } from '../../components/typography/BodyText';
+import { Title } from '../../components/typography/Title';
+import { PassportEvents } from '../../consts/analytics';
+import useHapticNavigation from '../../hooks/useHapticNavigation';
+import NFC_IMAGE from '../../images/nfc.png';
+import { ExpandableBottomLayout } from '../../layouts/ExpandableBottomLayout';
+import { black, slate100, white } from '../../utils/colors';
+
+interface PassportNFCScanScreenProps {}
+
+const PassportNFCScanScreen: React.FC<PassportNFCScanScreenProps> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)